tools/measurement/latency/src/test_target_id_map.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Map } from 'immutable';
+
+export const TEST_TARGET_ID_MAP: Map<string, number> = Map({
```

Comment:
Ideally, we should make this a file that's not embedded in the source code.

Reply:
Then will it also be checked into the repo?

Another reason to have it here is that prow jobs for many repos are running on the same cluster, and we need to have a way to tell whether a job is for Android SDK or other project.